frameworks/rooch-framework/sources/onchain_config.move

```diff
@@ -42,18 +44,33 @@ module rooch_framework::onchain_config {
         onchain_config().framework_version
     }
 
-    fun onchain_config_mut(): &mut OnchainConfig {
-        let object_id = object::named_object_id<OnchainConfig>();
-        let obj = object::borrow_mut_object_extend<OnchainConfig>(object_id);
-        object::borrow_mut(obj)
-    }
-
     public fun onchain_config(): &OnchainConfig {
         let object_id = object::named_object_id<OnchainConfig>();
         let obj = object::borrow_object<OnchainConfig>(object_id);
         object::borrow(obj)
     }
 
+    /******  APIs for update module publishing allowlist. ******/
+    public fun add_to_publishing_allowlist(account: &signer, publisher: address) {
+        let sender = signer::address_of(account);
+        assert!(sender == sequencer(), ErrorNotSequencer);
+        let allowlist = move_module::borrow_mut_allowlist();
+        move_module::add_to_allowlist(allowlist, publisher);
```

Comment:
`add_to_allowlist` requires the system reserved account, but the sequencer is not system reserved account.